src/mumble/GlobalShortcut.cpp

```diff
@@ -625,6 +625,27 @@ void GlobalShortcutConfig::on_qpbRemove_clicked(bool) {
 	qlShortcuts.removeAt(idx);
 }
 
+void GlobalShortcutConfig::on_qpbImport_clicked() {
+	commit();
+	QString filePath = QFileDialog::getOpenFileName(this, tr("Import shortcuts"), QString(), QLatin1String("*.ini"));
+	if (filePath.isEmpty()) {
+		return;
+	}
+	QSettings s(filePath, QSettings::IniFormat);
+	qlShortcuts = g.s.loadShortcuts(&s);
```

Comment:
Not ideal. If we are passed a non-existent / invalid file we will nuke the users shortcuts without being able to say what happened. Should at least try whether we can use the status() function to verify the ini didn't contain any obvious errors. To make it actually robust we probably have to write/load an additional key in the export/import to indicate this is a mumble shortcut file. That way the user cannot accidentally select some random ini and we can notify properly if we end up with an invalid file. If we make it a versiony thing it might come in handy in the future.

Reply:
The `SAVELOAD` makro uses `settings_ptr`…